packages/paste-core/components/alert/src/Alert.tsx

```diff
@@ -107,7 +112,20 @@ export interface AlertProps extends HTMLPasteProps<"div"> {
   element?: BoxProps["element"];
 }
 
-const renderAlertIcon = (variant: AlertVariants, element: string, title: string): React.ReactElement => {
+interface AlertIconProps {
+  variant: AlertVariants;
+  element: string;
+  title: string;
+}
+
+/**
+ * Component to display the appropriate alert icon.
+ */
+const AlertIcon: React.FC<AlertIconProps> = ({
+  variant,
+  element,
+  title,
+}): React.ReactElement => {
```

Comment:
Refactored this to be a component because it is a function that takes in args and returns JSX. It effectively had a signature similar to a React component.